eclipse/plugins/org.switchyard.tools.models.switchyard1_0/src/org/switchyard/tools/models/switchyard1_0/soap/impl/MessageComposerTypeImpl.java

```diff
@@ -187,7 +273,9 @@ public String toString() {
         if (eIsProxy()) return super.toString();
 
         StringBuffer result = new StringBuffer(super.toString());
-        result.append(" (unwrapped: ");
+        result.append(" (copyNamespaces: ");
+        if (copyNamespacesESet) result.append(copyNamespaces); else result.append("<unset>");
```

Comment:
I really prefer when there is a single statement per line
If on one line, for very simple case, I prefer using the copyNamespacesESet ? result.append(copyNamespaces) : result.append("<unset>") notation